core/modules/book/book.install

```diff
@@ -123,6 +123,15 @@ function book_update_1001() {
   $config->save();
 }
 
+/**
+ * Add option for reorder link.
+ */
+function book_update_1002() {
+  $config = config('book.settings');
+  $config->set('book_links', array('book_reorder'));
```

Comment:
Uh-oh, I don't think this will work as expected. If previously "book_add_child" was set, that one would get removed, and only book_reorder will be set after that.

Or is this by intention?

Reply:
Fixed